resource-managers/kubernetes/docker-minimal-bundle/src/main/docker/spark-base/Dockerfile

```diff
@@ -34,6 +35,6 @@ COPY dockerfiles/spark-base/entrypoint.sh /opt/
 
 ENV SPARK_HOME /opt/spark
 
-WORKDIR /opt/spark
+WORKDIR /opt/spark/work-dir
```

Comment:
I think this is fine, I'd rather have this than have the potential for overwriting data or getting surprising results in general.